testData/solutions/jaws.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
Unless you think otherwise, I believe this is not used and can safely be deleted.